app/config.json

```diff
@@ -0,0 +1,37 @@
+[
+	{
+		"id": 1,
```

Comment:
Not sure how it would be much more difficult. You still have the index of the array. The real difference change would be to how you reference in vue